### DISCUSSION TITLE
errors from noah.3.6 (LIS-public-7.3.3)

### DISCUSSION THREAD

**Discussion by jachungithub** [NASA-LIS/LISF#1075] [Q&A] [answered]
Hi,

Due to repeated errors, I set lis.config as a single processor run. However, the Error was same as the MPI run. LIS just stopped at Reading  max snow albedo map from ./lis_input.d01.nc (please refer to the attached file: lislog.0000).  When  "Albedo data source", "Max snow albedo data source", and "Greenness data source" were set to "none", LIS just stopped after ./MODEL_OUTPUT_LIST.TBL (i.e., line 45 in the lislog.0000). 
As shown in lis.config, my experiment is a global run with 1° resolution. Any helps would be greately appreciated. 

Thanks a lot,
[noah36_errors.zip](https://github.com/NASA-LIS/LISF/files/8583329/noah36_errors.zip)

**Comment by dmocko** [comment, marked as answer]
Hi @jachungithub ,

Please change this line in your lis.config file:

`Noah.3.6 restart output interval:         1mon  #1da`

Change it to:

`Noah.3.6 restart output interval:         1mo  #1da`

Please see Section 7.19 Defining a time interval - of the LIS Users' Guide for how to specify the times in the config file(s).

Your error message pointed to Line 59 of noah36_readcrd.F90, which is how I was able to pinpoint the typo in your lis.config.

```
Image              PC                Routine            Line        Source             
LIS                00000000017EE048  Unknown               Unknown  Unknown
LIS                000000000181B942  Unknown               Unknown  Unknown
LIS                000000000181A6C4  Unknown               Unknown  Unknown
LIS                000000000071A080  lis_timemgrmod_mp        3691  LIS_timeMgrMod.F90
LIS                000000000130AF27  noah36_readcrd_            59  noah36_readcrd.F90
LIS                00000000012F73B2  noah36_lsmmod_mp_         202  noah36_lsmMod.F90
```

**Reply by jachungithub** [reply]
Thanks a lot. I got some ideas to tackle the errors.
